.circleci/config.yml

```diff
@@ -34,3 +71,39 @@ jobs:
           command: bash gradlew publishToMavenLocal
       - store_test_results:
           path: test-results
+
+  integrationtest:
+    working_directory: ~/amplify-android-home
+    docker:
+      - image: circleci/android@sha256:5cdc8626cc6f13efe5ed982cdcdb432b0472f8740fed8743a6461e025ad6cdfc
+    environment:
+      JVM_OPTS: -Xmx3200m
```

Comment:
Minor: Would be nice to have this as a ENV variable or a configurable parameter to the workflow run instance especially when we get failures due to size limits.